backend/api/routes/users.py

```
@@ -0,0 +1,785 @@
+        status="success",
+        email=request.email,
+        credits_added=request.amount,
+        new_balance=new_balance,
+        message=message,
+    )
+
+
+@router.post("/admin/users/{email}/disable")
+async def disable_user(
+    email: str,
+    auth_data: Tuple[str, UserType, int] = Depends(require_admin),
```

Ruff (0.14.10)

Unused function argument: `auth_data`

(ARG001)